requirements.txt

```diff
@@ -1,5 +1,10 @@
 colorlog
 ecdsa
+oslo.i18n==1.5.0
+stevedore==1.5.0
+oslo.serialization==1.4.0
+debtcollector==0.3.0
+cliff==1.4.3
```

Comment:
Can we drop some of these dependencies maybe?.. The less we have, the better chances we will have to resolve conflicts in the future